build.sbt

```diff
@@ -1216,7 +1216,15 @@ intellij := {
         else replaceLibrary(res, s"$modName-deps", None, moduleDep(modName, jars))
     })
 
+    // I can't figure out how to keep the entity escapes for \n in the attribute values after this use of XML transform.
+    // Patching the original version back in with more brutish parsing.
+    val R = """(?ims)(.*)(<copyright>.*</copyright>)(.*)""".r
+    val oldContents = IO.read(ipr)
+    val R(_, withEscapes, _) = oldContents
```

Comment:
This fails when someone still has the old `ipr`.. Make it an option?

Reply:
I'll leave it up to you if you're motivated enough to improve the above, otherwise feel free to merge as-is :)